lib/main_micro_alp.pm

```diff
@@ -299,6 +302,22 @@ sub load_slem_on_pc_tests {
     } elsif (get_var('PUBLIC_CLOUD_UPLOAD_IMG')) {
         loadtest("boot/boot_to_desktop");
         loadtest("publiccloud/upload_image");
+        # AISTACK test verification
+        if (get_var('PUBLIC_CLOUD_AISTACK')) {
```

Comment:
you need to move this "if" out from `} elsif (get_var('PUBLIC_CLOUD_UPLOAD_IMG')) {` condition . 
```
} elsif (get_var('PUBLIC_CLOUD_UPLOAD_IMG')) {
        loadtest("boot/boot_to_desktop");
        loadtest("publiccloud/upload_image");
        # AISTACK test verification
} elsif (get_var('PUBLIC_CLOUD_AISTACK')) {  
```

variables like `PUBLIC_CLOUD_UPLOAD_IMG` , `PUBLIC_CLOUD_AISTACK`, `PUBLIC_CLOUD_CONTAINERS` , `PUBLIC_CLOUD_TOOLS_CLI` expected to be mutually exclusive and code should follow this rule

Reply:
fixed